app/models/vacancy.rb

```diff
@@ -233,30 +233,16 @@ def _geoloc
                       :expires_on, error_clash_behaviour: :omit_gov_uk_date_field_error
 
   scope :active, (-> { where(status: %i[published draft]) })
-  scope :applicable, (-> { applicable_by_date.or(applicable_by_time) })
-  scope :applicable_by_time, (-> { where("expiry_time IS NOT NULL AND expiry_time >= ?", Time.zone.now) })
-  scope :applicable_by_date, (-> { where("expiry_time IS NULL AND expires_on >= ?", Time.zone.today) })
+  scope :applicable, (-> { where("expiry_time >= ?", Time.zone.now) })
```

Comment:
What happens for vacancies where the expiry_time column is NULL?

Reply:
I just run this on production:

```ruby
Vacancy.where("expiry_time IS NULL AND expires_on IS NOT NULL").find_each { |v| v.update_column :expiry_time, v.expires_on }
```

we do not have vacancies with expiry_time nil anymore

Reply:
Cool I was wondering if you'd run something like that. Nice!